src/index.js

```diff
@@ -1,6 +1,7 @@
 import postcss from "postcss";
 import camelCase from "lodash.camelcase";
 import genericNames from "generic-names";
+import unquote from 'unquote';
```

Comment:
Could you remove the external dependency and write a function for this task? I'm planning to reduce the number of the dependencies to improve stability and maintanability of the package

Reply:
Sure, should i directly copy this library to implement this feature?

Reply:
Yep, this will do